src/__tests__/Header.spec.js

```diff
@@ -178,5 +187,13 @@ describe('Header Unit Tests', () => {
       expect(headerDiv.props().draggableHeaderCell).toBeUndefined();
       expect(headerDiv.props().getValidFilterValues).toBeUndefined();
     });
+
+    it('execute onCellClick event on cellMetaData', () => {
+      spyOn(testAllProps.cellMetaData, 'onCellClick');
+      const wrapper = renderComponent(testAllProps);
+      const headerDiv = wrapper.find('div');
+      headerDiv.simulate('click');
+      expect(testAllProps.cellMetaData.onCellClick).toHaveBeenCalled();
+    });
```

Comment:
You should be verifying the parameters of cellMetaData. Its important to verify that the selected idx and rowIdx is -1,-1